src/leap/common/events/zmq_components.py

```diff
@@ -14,60 +14,61 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
-
-
 """
 The server for the events mechanism.
 """
-
-
 import os
 import logging
 import txzmq
 import re
-import time
 
 from abc import ABCMeta
 
-# XXX some distros don't package libsodium, so we have to be prepared for
-#     absence of zmq.auth
 try:
     import zmq.auth
-    from zmq.auth.thread import ThreadAuthenticator
 except ImportError:
     pass
 
+from txzmq.connection import ZmqEndpoint, ZmqEndpointType
+
 from leap.common.config import flags, get_path_prefix
 from leap.common.zmq_utils import zmq_has_curve
 from leap.common.zmq_utils import maybe_create_and_get_certificates
 from leap.common.zmq_utils import PUBLIC_KEYS_PREFIX
 
+from leap.common.events.auth import TxAuthenticator, TxAuthenticationRequest
```

Comment:
This is going to import zmq.auth, which in file is imported in a try/except (I guess to work on envs without libsodium). Is not going to throw an exception in some platforms this import? If not, why do we keep zmq.auth in a try/except?